assets/model_monitoring/components/src/shared_utilities/span_tree_utils.py

```diff
@@ -101,7 +111,12 @@ def trace_id(self) -> str:
 
     @classmethod
     def create_node_from_dict(cls, span_node_dict: dict) -> "SpanTreeNode":
```

Comment:
can we rename it to _create_node_from_dict() if this is not intended to be exposed?

Reply:
It's fine to be exposed and could be used in some random cases to build nodes if we so need. The note is because I noticed in token stats v1 PR that Shreeya was using it to manually deserialize each node instead of using the `SpanTree.create_tree_from_json_string()` method which does it all for you.